lib/matplotlib/axes/_base.py

```diff
@@ -2510,21 +2668,21 @@ def autoscale_view(self, tight=None, scalex=True, scaley=True):
         x_stickies = y_stickies = np.array([])
         if self.use_sticky_edges:
             # Only iterate over axes and artists if needed.  The check for
-            # ``hasattr(ax, "lines")`` is necessary because this can be called
-            # very early in the axes init process (e.g., for twin axes) when
-            # these attributes don't even exist yet, in which case
+            # ``hasattr(ax, "_children")`` is necessary because this can be
+            # called very early in the axes init process (e.g., for twin axes)
+            # when these attributes don't even exist yet, in which case
             # `get_children` would raise an AttributeError.
             if self._xmargin and scalex and self._autoscaleXon:
                 x_stickies = np.sort(np.concatenate([
```

Comment:
I find this 5 lines of code completely incomprehensible.  having two for statements and an if statement in a list comprehension is too cute in my opinion.